bgpd/bgp_ecommunity.h

```diff
@@ -95,6 +100,11 @@ struct ecommunity {
 	/* Reference counter.  */
 	unsigned long refcnt;
 
+	/* Size of Each Unit of Extended Communities attribute.
+	 * to differentiate between IPv6 ext comm and ext comm
+	 */
+	int unit_size;
```

Comment:
nit: it holds only 8 or 20, right? maybe an uint8_t or a bool with inlined function makes sense for this as well?

Reply:
ok

Reply:
ok